lib/build-config.js

```diff
@@ -21,7 +21,17 @@ function translateBundlerConfigs(bundleConfigs, options) {
 	return bundleConfigs.map(bundleConfig => {
 		let inputFileName = path.resolve(configDir, bundleConfig.entryPoint);
 		let outputStyle = compact ? "compressed" : "nested";
-		let includePaths = [ path.resolve(configDir, "node_modules") ];
+		// Use the node dependency resolution algorithm to figure out the
+		// includePaths – only available in Node 8.9 and up
+		let includePaths;
+		if(require.resolve.paths) {
+			includePaths = require.resolve.paths("something");
+		} else {
+			includePaths = [ path.resolve(configDir, "node_modules") ];
+			console.warn("You are using Node.js < 8.9. We can" +
+				"therefore only offer a fallback implementation for importing" +
+				"Sass files from NPM packages");
```

Comment:
* https://twitter.com/seldo/status/850042251635867648
* `+` doesn't insert a space in between strings 😃